examples/deadline/Local-Zone/README.md

```diff
@@ -0,0 +1,41 @@
+# RFDK Sample Application - Local Zones
+
+If you have large asset files that your worker instances need to process, any bit of latency can have a big impact on the time your renders take. This example will walk you through setting up your workers in a local zone while leaving the rest of the render farm in standard availability zones. Currently Amazon has launched a local zone in Los Angeles that is a part of the us-west-2 region, but they have more on the way. For more information on where local zones are avialable, how to get access, and what services they provide, refer to the [AWS Local Zones about page](https://aws.amazon.com/about-aws/global-infrastructure/localzones/).
```

Comment:
I think we'd want to add that the use-case for the local zone would be where you are connecting your Workers to a datacenter, or on-premises network, via Direct Connect or VPN and want to have your workers directly mount your filesystem over that connection. If the filesystem is closer to the local zone than other AWS regions, then the improved latency will improve render times.

This just says that latency can have a big impact, but doesn't draw the line from the on-prem/datacenter filer to wanting to use a local zone.

Also, are we just looking at Workers in the local zone? Is it possible to host more of the farm infrastructure in the local zone? e.g. The EFS & DocDB for the repo, and/or the RenderQueue?

Reply:
EFS and DocDB don't have support in the local zone. I could look at moving the Render Queue, but I'm not sure what benefit there would be since the repository would still be in a standard availability zone.

Reply:
The use-case is now documented with some guidance on how to go about connecting infrastructure outside of AWS. Resolving